guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -4,11 +4,13 @@
 Use the following procedure to register a host to {ProjectNameX}.
 
 With this registration method you can register hosts to {Project} by generating a `curl` command on {Project} and running this command on hosts.
-This method uses the global registration template that gives you complete control over the process of the host registration.
+This method uses two templates: `global registration` template and `host initial configuration` template. That gives you complete control over the process of the host registration.
+Default templates can be set in the Settings > provisioning.
+
 
 ifndef::satellite[]
 Note that you can extend the parameters by plug-ins.
-For more information, see https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc[How to Create a Plugin].
+For more information, see https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc[How to Create a Plugin] and http://theforeman.github.io/foreman/?path=/docs/introduction-slot-and-fill--page.asciidoc[Slot and Fill]
```

Comment:
Nitpick: the URL does not open the correct chapter, but it opens the page.

Reply:
Really? For me it opens _Slot and Fill_ page correctly: http://theforeman.github.io/foreman/?path=/docs/introduction-slot-and-fill--page